lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ReorderProducer.java

```diff
@@ -249,7 +249,7 @@ public void fillComponents(UIContainer tofill, ViewParameters params, ComponentC
 				    // oembed. use the URL
 				    UILink.make(row, "link", i.getAttribute("multimediaUrl"), i.getAttribute("multimediaUrl"));
 				} else if (i.getType() == SimplePageItem.QUESTION) {
-				    String text = i.getAttribute("questionText");
+				    String text = i.getAttribute("questionCorrectText");
```

Comment:
what is questionCorrectText?

Reply:
`questionCorrectText` is the original question text without the `<p>` tag, The `<p>` gets added from the Iframe I think.  The span tag won't work with `<p>`:  https://forum.bootstrapstudio.io/t/unable-to-add-p-tags-inside-a-span-tag/6149

Reply:
![24](https://github.com/sakaiproject/sakai/assets/50500283/5672bdb7-bfb3-4681-9e12-8950f1d84269)

Reply:
are you sure?!

public String questionText, questionCorrectText, questionIncorrectText;

i assume questionCorrect and questionIncorrect refer to correct and incorrect feedback

Reply:
Please let me check.